eclair/src/diff.rs

```diff
@@ -80,14 +178,21 @@ pub fn diff(candidate: &Summary, reference: &Summary) {
 
         let y_title = format!("{} [{}]", id.name, reference.items[index].unit);
 
-        let layout = Layout::new()
-            .height(800)
-            .title(Title::new(title.as_str()))
-            .x_axis(Axis::new().title(Title::new("Date")))
-            .y_axis(Axis::new().title(Title::new(y_title.as_str())));
-        plot.set_layout(layout);
-
-        println!("{}", plot.to_inline_html(Some("My plot")));
-        break;
+        common_plots.push(UPlotChart::new(&merged_time, padded_data, title, y_title));
     }
+
+    let diff_html = DiffTemplate {
+        candidate_name: &candidate.name,
+        reference_name: &reference.name,
+        common_plots,
+    };
+
+    let rendered = diff_html.render().unwrap();
+
+    let mut file = if let Some(name) = output {
+        File::create(name).unwrap()
+    } else {
+        File::create("diff.html").unwrap()
+    };
```

Comment:
You could push this `if` condition into the argument position if you wanted:

<suggestion>
```rust
    let mut file = 
        File::create(if let Some(name) = output {
            name
        } else {
            "diff.html"
        }).unwrap();
```
</suggestion>